code/modules/paperwork/photocopier.dm

```diff
@@ -9,15 +9,49 @@
 /// How much toner is used for making a copy of a photo.
 #define PHOTO_TONER_USE 0.625
 /// How much toner is used for making a copy of a document.
-#define DOCUMENT_TONER_USE 0.75
+#define DOCUMENT_TONER_USE (PAPER_TONER_USE * DOCUMENT_PAPER_USE)
 /// How much toner is used for making a copy of an ass.
-#define ASS_TONER_USE 0.625
-/// How much toner is used for making a copy of paperwork
-#define PAPERWORK_TONER_USE 0.75
-
+#define ASS_TONER_USE PHOTO_TONER_USE
+/// How much toner is used for making a copy of paperwork.
+#define PAPERWORK_TONER_USE (PAPER_TONER_USE * PAPERWORK_PAPER_USE)
+
+/// At which toner charge amount we start losing color. Toner cartridges are scams.
+#define TONER_CHARGE_LOW_AMOUNT 2
+
+// please use integers here
+/// How much paper is used for making a copy of paper. What, are you seriously surprised by this?
+#define PAPER_PAPER_USE 1
+/// How much paper is used for making a copy of a photo.
+#define PHOTO_PAPER_USE 1
+/// How much paper is used for making a copy of a document.
+#define DOCUMENT_PAPER_USE 20
+/// How much paper is used for making a copy of a photo.
+#define ASS_PAPER_USE PHOTO_PAPER_USE
+/// How much paper is used for making a copy of paperwork.
+#define PAPERWORK_PAPER_USE 10
+
+/// Maximum capacity of a photocopier
+#define MAX_PAPER_CAPACITY 60
 /// The maximum amount of copies you can make with one press of the copy button.
 #define MAX_COPIES_AT_ONCE 10
 
+/// Photocopier copy fee.
+#define PHOTOCOPIER_FEE 5
+
+///Paper blanks (form templates, basically). Loaded from `config/blanks.json`.
+///If invalid or not found, set to null.
+GLOBAL_LIST_INIT(paper_blanks, init_paper_blanks())
+
+/proc/init_paper_blanks()
```

Comment:
do we really need `try/catch` stuff for this? couldn't we just length-check `blanks_json` after we do the file access and then return if we don't have anything?

Reply:
it just feels un-necessary, lmk if you have a concrete reason as to why it's needed _now_, i see you got it from the old code but that was probably written like that since it was in that hot UI-code, but this literally just needs to run one time to init the global.

Reply:
no, I got it from the old code, and since I don't know if json decode or readfile throws which could break something, I just left it in there